joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/TypeMap.scala

```diff
@@ -100,8 +100,8 @@ class TypeMap(astGenResult: AstGenRunnerResult, initialMappings: List[NamespaceT
       .map(AstCreatorHelper.createDotNetNodeInfo(_))
       .filter { x =>
         x.node match
-          case NamespaceDeclaration | FileScopedNamespaceDeclaration | ClassDeclaration => true
-          case _                                                                        => false
+          case NamespaceDeclaration | FileScopedNamespaceDeclaration | ClassDeclaration | InterfaceDeclaration => true
```

Comment:
Perhaps it would make sense to bind these together under a single trait, something like `NamespaceOrTypeDeclaration`. Then we could add `Enum` and `Record` too